src/web/mjs/connectors/templates/Tappytoon.mjs

```diff
@@ -68,9 +68,15 @@ export default class Tappytoon extends Connector {
     }
 
     async _getPages(chapter) {
-        let uri = new URL(`/chapters/${chapter.id}?includes=images`, this.apiurl);
+        //check if super quality is supported
+        let uri = new URL(`/comics/${chapter.manga.id}`, this.apiurl);
         let request = new Request(uri, this.requestOptions);
         let data = await this.fetchJSON(request);
-        return data.images.map(image => image.url);
+        const quality = data.isSuperHighQualitySupported ? 'super_high': 'high';
+        uri = new URL(`/content-delivery/contents?chapterId=${chapter.id}&variant=${quality}&locale=${this.lang}`, this.apiurl);
```

Comment:
It seems `locale` parameter can be omitted since `this.lang` is never set to something other than `undefined`

Reply:
```this.lang``` is always defined in connectors using this template. 

https://github.com/manga-download/hakuneko/blob/e9d2abe4f18f9576ae4270e4889bb8ef1f6a0006/src/web/mjs/connectors/TappytoonDE.mjs#L15

https://github.com/manga-download/hakuneko/blob/e9d2abe4f18f9576ae4270e4889bb8ef1f6a0006/src/web/mjs/connectors/TappytoonEN.mjs#L15

https://github.com/manga-download/hakuneko/blob/e9d2abe4f18f9576ae4270e4889bb8ef1f6a0006/src/web/mjs/connectors/TappytoonFR.mjs#L15

Nothing to see here ;).